frontend/integration-tests/tests/kubevirt/vm.overview.scenario.ts

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-undef */
+import { browser, ExpectedConditions as until } from 'protractor';
+
+import { appHost, testName } from '../../protractor.conf';
+import { isLoaded } from '../../views/crud.view';
+import { basicVmConfig, cloudInitCustomScriptConfig } from './mocks';
+import { statusIcon, statusIcons, overviewTab } from '../../views/kubevirt/virtualMachine.view';
+import { removeLeakedResources, VM_BOOTUP_TIMEOUT } from './utils';
+import Wizard from './models/wizard';
+import { VirtualMachine } from './models/virtualMachine';
+
+describe('Test VM overview page', () => {
+  const leakedResources = new Set<string>();
+  const wizard = new Wizard();
+  const vmName = `vm-${testName}`;
+  const vm = new VirtualMachine(vmName, testName);
+
+  afterAll(async() => {
+    removeLeakedResources(leakedResources);
+  });
+
+  it('Creates VM from wizard', async() => {
```

Comment:
should the VM be created from the wizard? Shouldn't this test only the vm overview page if it's called `vm.overview.scenario.ts` ?

Reply:
how do you think if putting it in `beforeAll`?

Reply:
I was intending to use UI as much as possible. But I have no problem to create it from yaml.

Reply:
I move it to `beforeAll`. If the wizard broke someday, let's fixing both the wizard and the test.

Reply:
I meant creating it directly from a mock/fixture instead of the UI. @rhrazdil what is your opinion on this?

Reply:
I agree that the VM is very generic, so there is no benefit in using the wizard. Probably better save some time and create it from yaml. 

You can use the existing getVmManifest method, we just need to extend it to support cloudinitdisk, you can do that by adding optional parameter ```cloudinit?: string``` and on line 78 put something like:
```
if (cloudinit) {
    volumes.push({
      cloudInitNoCloud: {
        userData: cloudinit,
      },
      name: 'cloudinitdisk',
    });
  }
```

Then, you can create the VM by:
```
execSync(`echo '${JSON.stringify(getVmManifest('Container', testName, vmName, cloudinit))}' | kubectl create -f -`);
```
I haven't tested it, so there might be some bug, let me know if you have an issue.

Reply:
updated.